tfe/data_source_workspace_ids.go

```diff
@@ -42,6 +50,10 @@ func dataSourceTFEWorkspaceIDsRead(d *schema.ResourceData, meta interface{}) err
 	// Get the organization.
 	organization := d.Get("organization").(string)
 
+	if len(d.Get("names").([]interface{})) == 0 && len(d.Get("tags").([]interface{})) == 0 {
+		return nil
```

Comment:
Shouldn't this return an error?

Reply:
Definitely. I think I meant to look up how errors worked later. I'll add that now.